installer.sh

```diff
@@ -182,8 +182,11 @@ esac
 # Command line params
 STUB=0
 KEYLIME_DIR=
-while getopts ":shckmp:" opt; do
+# LISTEN_ALL_IPS=1 means than will set the binding and listening IP about the agent, verifier, and registrar server as 0.0.0.0
+LISTEN_ALL_IPS=0
+while getopts ":shcikmp:" opt; do
```

Comment:
`s` and `c` are not use (anymore), so they could be removed.

Reply:
Thanks @stefanberger , I fixed it.